android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java

```diff
@@ -37,12 +38,14 @@
  * WebView instance.
  */
 public class BridgeWebChromeClient extends WebChromeClient {
-    private Bridge bridge;
     static final int FILE_CHOOSER = PluginRequestCodes.FILE_CHOOSER;
     static final int FILE_CHOOSER_IMAGE_CAPTURE = PluginRequestCodes.FILE_CHOOSER_IMAGE_CAPTURE;
     static final int FILE_CHOOSER_VIDEO_CAPTURE = PluginRequestCodes.FILE_CHOOSER_VIDEO_CAPTURE;
     static final int FILE_CHOOSER_CAMERA_PERMISSION = PluginRequestCodes.FILE_CHOOSER_CAMERA_PERMISSION;
     static final int GET_USER_MEDIA_PERMISSIONS = PluginRequestCodes.GET_USER_MEDIA_PERMISSIONS;
+    static final int GEOLOCATION_REQUEST_PERMISSIONS = PluginRequestCodes.GEOLOCATION_REQUEST_PERMISSIONS;
```

Comment:
Shouldn't use the same request code that is used on Geolocation plugin, the codes are used to identify who called the permission prompt, so if the Geolocation plugin is installed it could cause an undesired callback.